### DISCUSSION TITLE
Clone block error

### DISCUSSION THREAD

**Discussion by Faris90** [Alessandrito123/Snavanced#8] [Errors and Bugs]
Self explanitory
![Untitled script pic(2)](https://github.com/Alessandrito123/Super-Snap/assets/24720470/442ca6a0-d56f-40e0-a8a2-b55c4c2a35be)

**Comment by Alessandrito123** [comment]
Thank you for reporting the bug, eman is the newName variable, for shorting. I updated it recently. :~)
